### ISSUE TITLE
Offset between picked mouse location and point cloud in version 2.11 snap package.

### ISSUE THREAD

**Issue by mevatron** [CloudCompare/CloudCompare#1293]
**Describe the bug**
Using the 2.11 snap package on KDE Neon, I'm getting a large offset from the picked point location shown in the GIF below:

![cc-2 11-pick-offsets](https://user-images.githubusercontent.com/714585/86991574-0e0fde00-c165-11ea-9bad-bf1112dbc041.gif)

I was not seeing this behavior in the previous snap package, but it doesn't appear to be available for downgrade :(

**How to reproduce**

**Expected behaviour**
Picked locations should align to point nearest the mouse pointer.

**Your environment**
 - CC Version (or git hash/tag): snap package latest/stable: 2.11 2020-06-14
 - OS & Version: KDE Neon Linux
 - Graphics card: Intel
 - Qt Version (if compiling):  N/A

**Comment by rjanvier** [comment]
ho, sorry I didn't promote the last package to stable channel. It should be fixed right now. Could you try to update and test again? thanks

**Comment by dgirardeau** [comment]
Actually I believe this issues simply comes from the fact that you have loaded a cloud with large coordinates **without applying a global shift**!

There's big loss of accuracy that applies both on the coordinates themselves, but also at display time (as OpenGL doesn't like that as well). I bet te

**Comment by rjanvier** [comment]
yes, Daniel is maybe right, I didn't look at your print screen and we fix a similar issue just before 2.11 (but for meshes)

**Comment by mevatron** [comment]
@dgirardeau @rjanvier Thanks for quick responses! You are correct that the point values quite large, but I usually accept the defaults shift the points. Below shows the defaults I get when opening this particular point cloud:

![image](https://user-images.githubusercontent.com/714585/87070944-27ee0700-c1df-11ea-870e-a33f80703652.png)

Is a shift not enough in this case? Is scale also required? I've usually just accept the default in previous versions, but maybe something changed there?

**Comment by dgirardeau** [comment]
Oh you are right, this cloud was recentered apparently! But then it has all the characteristics of a cloud that has lost its accuracy... Was it saved like that previously by mistake maybe?

**Comment by mevatron** [comment]
So, I figured out how to revert to the previous snap with `snap revert cloudcompare`, and it does look like some kind of bug in 2.11 as here is what I see on the same point cloud with 2.10.1:
![cc-2 10 1-pick-offsets](https://user-images.githubusercontent.com/714585/87098218-677f1800-c20c-11ea-8557-f512a2b881dc.gif)

If there's any other tests I can run to be of help, I'd be happy to try them out!

**Comment by rjanvier** [comment]
I am unable to reproduce this with few of my clouds. Could you please send us even a cropped version of yours in order to investigate on our side?

**Comment by mevatron** [comment]
Sure can! I snipped out the building to make the point cloud smaller. I also test 2.11 and 2.10.1 on it just to be sure the behavior is the same on the cropped cloud and it is still giving me offset picks on 2.11.

https://drive.google.com/file/d/1PlepvjZSB7ZC99bkwXWZftj8NfV4-Sog/view?usp=sharing

**Comment by rjanvier** [comment]
Thanks, just tested and (un)fortunately I can't reproduce it.

**Comment by dgirardeau** [comment]
I tested on my side as well (Windows) and had no issue. Which picking mode are you using? (GPU or CPU)?

**Comment by mevatron** [comment]
@rjanvier Interesting, maybe it's something to do with KDE Neon's Qt version? I'll have to try building with source to see if that does anything different.

@dgirardeau How do you select the picking mode? I usually just click the point list mode on the toolbar, so I guess whatever the default is there?

**Comment by dgirardeau** [comment]
CloudCompare asks it the first time and depending on your choice it might remember that for good.

You can look at the 'Display Settings' (Display > Display parameters). It the last option ('Auto-compute octree for picking' - I now realize the choice is not between CPU or GPU but between octree or brute force now ;).

**Comment by rjanvier** [comment]
@mevatron, yes you can try building from source to see if it is related somehow with the Snap (Snap come with its own QT version). What is your ditrib name and version? Is your screen an HDPI one? thanks.

**Comment by mevatron** [comment]
@rjanvier I haven't had a chance to build from source yet, but I will try that later this week. 

Below is my `/etc/lsb-release` file (KDE Neon is based on Ubuntu 18.04, but they usually utilize the very latest Qt version):

```
DISTRIB_ID=neon
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="KDE neon User Edition 5.19"
```
My screen is just a normal 1080p monitor for both the laptop screen and external monitor.

**Comment by rjanvier** [comment]
Hi, sorry but it's unlikely you'll had a future update on this issue. 
Snap is no longer supported by the community, we fully move our effort to flatpak packaging.

A 2.13 beta version is available on Flathub
Please read [this](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111) if you want to know the basic about beta channel on flathub. For CC, basically you have to do:

```
flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta org.cloudcompare.CloudCompare
```

You may need to make the beta version the default one with

```
flatpak make-current org.cloudcompare.CloudCompare beta
```

**Comment by rjanvier** [comment]
Hi this should
